clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1905,6 +1905,16 @@ bool CompilerInvocation::ParseCodeGenArgs(CodeGenOptions &Opts, ArgList &Args,
     Opts.setInlining(CodeGenOptions::NormalInlining);
   }
 
+  // -mcmodel option.
+  if (const llvm::opt::Arg *A =
+          Args.getLastArg(clang::driver::options::OPT_mcmodel_EQ)) {
+    llvm::StringRef modelName = A->getValue();
+    if (modelName == "tiny" && !(T.isARM() || T.isAArch64())) {
+      Diags.Report(diag::err_drv_unsupported_option_argument_for_target)
+          << A->getSpelling() << modelName << T.getTriple();
```

Comment:
```suggestion
    StringRef ModelName = A->getValue();
    if (ModelName == "tiny" && !(T.isARM() || T.isAArch64())) {
      Diags.Report(diag::err_drv_unsupported_option_argument_for_target)
          << A->getSpelling() << ModelName << T.getTriple();
```
Fixing for our coding style.